core_v2/sources/domains.move

```diff
@@ -60,6 +60,34 @@ module aptos_names_v2::domains {
         capability: SignerCapability,
     }
 
+    /// Manager object refs
+    struct Manager has key {
+        /// The extend_ref of the manager object to get its signer
+        extend_ref: object::ExtendRef,
+    }
+
+    #[resource_group_member(group = aptos_framework::object::ObjectGroup)]
+    struct DomainNameRecordTokenV2 has key {
+        name_domain_name: String,
+        expiration_time_sec: u64,
+        target_address: Option<address>,
+        subdomain_collection_name: String,
+
+        extend_ref: object::ExtendRef,
+        transfer_ref: object::TransferRef,
+    }
+
+    #[resource_group_member(group = aptos_framework::object::ObjectGroup)]
+    struct SubdomainNameRecordTokenV2 has key {
+        name_subdomain_name: String,
+        expiration_time_sec: u64,
```

Comment:
we need a expiration flag as `auto-renewal` or a better name here. 
when the flag is true, we will refer the expiration date of the domain as the expiration date for this subdomain

Reply:
i name it `use_domain_expiration_time`